### PR TITLE
docs(readme): correct wording and grammar in README_de_DE.md

### PR DESCRIPTION
**Pull request by traktuner**
This PR fixes grammar mistakes and enhaces the wording of some phrases.
Also, the GitHub donations links at the bottom reflect the recent changes to support "immich-app" instead of "alextran1502".

### CODE REVIEW

File: README_de_DE.md [hunk 1]
```diff
@@ -101,22 +101,22 @@ Spec: Free-tier Oracle VM - Amsterdam - 2.4Ghz quad-core ARM64 CPU, 24GB RAM
 
 ## Unterstütze das Projekt
 
-Ich habe mich diesem Projekt verpflichtet und werde nicht aufgeben. Ich werde die Dokumentation weiter aktualisieren, neue Funktionen hinzufügen und Fehler beheben. Allerdings kann ich das nicht alleine schaffen. Daher brauche ich Eure Unterstützung mir zusätzliche Motivation zu geben um weiterzumachen.
+Ich habe mich diesem Projekt verpflichtet und werde nicht aufgeben. Ich werde die Dokumentation weiter aktualisieren, neue Funktionen hinzufügen und Fehler beheben. Allerdings kann ich das nicht alleine schaffen. Daher brauche ich eure Unterstützung, um mir zusätzliche Motivation zu geben, weiterzumachen.
```
Comment (danieldietzler) [inline README_de_DE.md:104]: 'eure' could and imo should be upper case.
> In allen Textformen, in denen der Autor selbst den Leser persönlich anredet, kann man du/Du und die genannten Wörter großschreiben.
(duden.de)

File: README_de_DE.md [hunk 2]
```diff
@@ -101,22 +101,22 @@ Spec: Free-tier Oracle VM - Amsterdam - 2.4Ghz quad-core ARM64 CPU, 24GB RAM
...
 
-Wie unsere Gastgeber in der [selfhosted.show - In der Episode 'The-organization-must-not-be-name is a Hostile Actor'](https://selfhosted.show/79?t=1418) gesagt haben, ist dies ein riesiges Unterfangen, welchem das Team und ich uns annehmen. In Zukunft würde ich liebend gerne Vollzeit an dem Projekt arbeiten und bitte daher um Eure Unterstützung.
+Wie unsere Gastgeber in der [selfhosted.show - In der Episode 'The-organization-must-not-be-name is a Hostile Actor'](https://selfhosted.show/79?t=1418) gesagt haben, ist dies ein riesiges Unterfangen, welchem das Team und ich uns annehmen. In Zukunft würde ich liebend gerne Vollzeit an dem Projekt arbeiten und bitte daher um eure Unterstützung.
```
Comment (danieldietzler) [inline README_de_DE.md:106]: Same here.

File: README_de_DE.md [hunk 3]
```diff
@@ -101,22 +101,22 @@ Spec: Free-tier Oracle VM - Amsterdam - 2.4Ghz quad-core ARM64 CPU, 24GB RAM
...
 
-Wenn Du denkst, dass dies die richtige Sache ist und dich selbst die App für eine längere Zeit nutzen siehst, dann denke bitte darüber nach das Projekt mit einer der unten aufgelisteten Optionen zu unterstützen.  
+Wenn du denkst, dass dies die richtige Sache ist und dich selbst die App für eine längere Zeit nutzen siehst, dann denke bitte darüber nach, das Projekt mit einer der unten aufgelisteten Optionen zu unterstützen.
```
Comment (danieldietzler) [inline README_de_DE.md:108]: And here.

**Review by danieldietzler — CHANGES_REQUESTED** [review]

**Comment by traktuner** [comment]
Thanks, i corrected those. Also I changed "Du" to uppercase in the ##Demo section so that it's consistent everywhere

**Review by danieldietzler — APPROVED** [review]
Thanks a lot! :)

**Pull request merged**